.gitattributes

```diff
@@ -1,7 +1,10 @@
+* text=auto eol=lf
+*.bat eol=crlf
+
 *.v linguist-language=V text=auto eol=lf
 *.vv linguist-language=V text=auto eol=lf
 *.vsh linguist-language=V text=auto eol=lf
 **/v.mod linguist-language=V text=auto eol=lf
-*.bat text=auto eol=crlf
- Dockerfile.* linguist-language=Dockerfile
-*.toml text eol=lf
```

Comment:
why is the *.toml entry removed?

Reply:
since `* text=auto eol=lf` which we also add during default scaffolding is added.